packages/provider-url-service/.firebaserc

```diff
@@ -1,6 +1,7 @@
 {
   "projects": {
     "alfajores": "celo-mobile-alfajores",
-    "mainnet": "celo-mobile-mainnet"
+    "mainnet": "celo-mobile-mainnet",
+    "testnet": "celo-testnet-production"
```

Comment:
Just double checking since seeing testnet production looks a bit weird to me, although it's probably intended

Reply:
Yeah unfortunately we must deploy to this because that's where all our data is stored. I couldn't find an easy way to read from a different project environment than the one you're deployed to without exposing some sensitive data

Reply:
Nvm figured this out!